### ISSUE TITLE
Security Issue (Mail-Spoofing)

### ISSUE THREAD

**Issue by mmundi72** [danmunn/redmine_dmsf#821]
We host our Redmine installation at an external service provider. We have about 200 redmine users from different companies. 
Due to some mail spoofing attacks (meaning: mails have been sent using a mailadress from our company but the mail has been sent from an external mail server) the policies have been changed:

Let's assume our domain is "mycompany.com" and all mailadresses are like  name@mycompany.com

Sending a document (or reference thereof) from within DMSF-Plugin is not possible if I send from name@company.com to second@company.com  (because the policy denies it). It's not a problem to sent the mail to any other domain.

As there are different companies invovled it could happen to any of these companies.

Therefore I would like to have to following option:

- Option to choose from:
-- "sender" is currently logged in user (eg. name@company.com) 
-- general mailadresse (eg.  no-reply@no-reply.com)
-- define "reply-to"-address (especiall for the general mailaddress case it is helpful to set the reply-to-address to the mailadress of the currently logged in user

I'm not sure if this is already possible - if not it would be great to have this important enhancement.

**Comment by picman** [comment]
Related to #708

**Comment by picman** [comment]
There is **Emission email address** option in the Redmine Settings. You can change it there, or?

**Comment by mmundi72** [comment]
Yes, but DMSF does not send mails from this address. Our setting is as follows:

![emssion-email-address](https://user-images.githubusercontent.com/7224922/35155581-5e0c02ec-fd2e-11e7-9a32-a0abf49f13dd.jpg)

All mails from Redmine are coming from the above mail address (any change to tickets, wiki pages...)

But DMSF does us my personal mail address as sender:

![dmsf-email-sender](https://user-images.githubusercontent.com/7224922/35155603-73788dbc-fd2e-11e7-9b3a-3f650b514b20.jpg)
